cve_bin_tool/cli.py

```diff
@@ -170,19 +160,22 @@ def main(argv=None):
         help="comma-separated list of checkers to enable",
     )
 
-    try:
+    with ErrorHandler(mode=ErrorMode.NoTrace):
         args = parser.parse_args(argv[1:])
-    except SystemExit:
-        # override default argparse exit(2) behaviour so positive numbers can indicate cves
-        parser.print_help()
-        return -2
 
     if args.log_level:
         LOGGER.setLevel(args.log_level)
 
     if args.quiet:
         LOGGER.setLevel(logging.CRITICAL)
 
+    if 0 < LOGGER.level <= 10:
+        error_mode = ErrorMode.FullTrace
+    elif LOGGER.level >= 50:
+        error_mode = ErrorMode.NoTrace
+    else:
+        error_mode = ErrorMode.TruncTrace
+
```

Comment:
Set error mode according to logging level. It will not print exception trace in quite mode (Currently we are printing that)